CHANGELOG.md

```diff
@@ -20,26 +20,47 @@ argument or resolve the type ambiguity by casting to the intended type.
 **Bugfixes**
 * #4619 Improve handling enum columns in compressed hypertables
 * #4673 Fix now() constification for VIEWs
-* #4676 Fix a deadlock when decompressing chunks and performing SELECTs
 * #4681 Fix compression_chunk_size primary key
-* #4685 Improve chunk exclusion for space dimensions
 * #4696 Report warning when enabling compression on hypertable
+* #4745 Fix FK constraint violation error while insert into hypertable which references partitioned table
+* #4756 Improve compression job IO performance
+
+**Thanks**
+* @jvanns for reporting hypertable FK reference to vanilla PostgreSQL partitioned table doesn't seem to work
```

Comment:
```suggestion
```
This needs to go into 2.8.1 section since it belongs to #2658

Reply:
@svenklemm This belongs to #4745, which is not included so far (the PR was merged after I created the initial list of commits that should be included in the release). Are you in favor of including the #4745 PR also in the 2.8.1?